framework/db/oci/Schema.php

```diff
@@ -219,7 +215,15 @@ protected function createColumn($column)
             if (stripos($column['DATA_DEFAULT'], 'timestamp') !== false) {
                 $c->defaultValue = null;
             } else {
-                $c->defaultValue = $c->phpTypecast($column['DATA_DEFAULT']);
+                $defaultValue = $column['DATA_DEFAULT'];
+                if ($defaultValue !== null) {
+                    if (($len=strlen($defaultValue)) > 2 && $defaultValue[0] === "'" && $defaultValue[$len-1] === "'") {
+                        $defaultValue = substr($column['DATA_DEFAULT'], 1, -1);
```

Comment:
Why? It's the default value expression from the db schema, not user input. Examples:

* numerics: `12.34`
* strings: `'something'`
* expressions: `to_timestamp('value','format')` and `CURRENT_TIMESTAMP`.